src/apps/ScopeChangeRequest/workspaceConfig/filter/filterConfig.ts

```diff
@@ -1,4 +1,5 @@
 import { FilterOptions, FilterValueType } from '../../../../packages/Filter/Types';
+import { Contributor, WorkflowStep } from '../../../../packages/Workflow/src';
```

Comment:
```suggestion
import { WorkflowStep } from '@equinor/Workflow';
```

Du bruker ikke Contributor her. Også ser det finere ut når vi tar i bruk alias import (`@equinor/Workflow`) istedenfor `../../../../packages/Workflow/src`